src/components/Menu/Menu.tsx

```diff
@@ -13,6 +13,8 @@ type MenuProps = {
   menuHref: LinkProps['href'];
 };
 
+const invalidChildren = ["/[platform]/prev"];
```

Comment:
The child node found with this route will be excluded from the menu

Reply:
Is this because `/prev` shouldn't be an actual page?

Reply:
What this does is excludes the `/[platform]/prev` child (and its children) from being included as part of another menu.  This is because `/[platform]/prev` should either be the root node or not appear at all, I don't think there is currently a case where we want it to appear as a child in the menu